File: benchmark/lua/sum_of_squares.lua
local sum = 0
local start = os.clock()
for i = 1, 1000000 do
    sum = sum + i * i
end
local finish = os.clock()
print("Sum: " .. sum)
print(string.format("Time: %.0fms", (finish - start) * 1000))

File: benchmark/perl/sum_of_squares.pl
use Time::HiRes qw(time);
my $start = time();
my $s = 0;
for my $i (1..1000000) { $s += $i * $i; }
my $end = time();
my $ms = ($end - $start) * 1000;
print "Sum: $s
";
printf "Time: %.0fms
", $ms;
